core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/spreadsheet/LibraryDilutionSpreadSheets.java

```diff
@@ -0,0 +1,71 @@
+package uk.ac.bbsrc.tgac.miso.core.data.spreadsheet;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import uk.ac.bbsrc.tgac.miso.core.data.DetailedSample;
+import uk.ac.bbsrc.tgac.miso.core.data.Index;
+import uk.ac.bbsrc.tgac.miso.core.data.SampleIdentity;
+import uk.ac.bbsrc.tgac.miso.core.data.SampleTissue;
+import uk.ac.bbsrc.tgac.miso.core.data.impl.LibraryDilution;
+import uk.ac.bbsrc.tgac.miso.core.util.BoxUtils;
+import uk.ac.bbsrc.tgac.miso.core.util.LimsUtils;
+
+public enum LibraryDilutionSpreadSheets implements Spreadsheet<LibraryDilution> {
+  TRACKING_LIST("Tracking List", //
+      Column.forString("Name", LibraryDilution::getName), //
+      Column.forString("Barcode", LibraryDilution::getIdentificationBarcode), //
+      Column.forString("Library Name", dilution -> dilution.getLibrary().getName()), //
+      Column.forString("Library Alias", dilution -> dilution.getLibrary().getAlias()), //
+      Column.forString("Library Type", dilution -> dilution.getLibrary().getLibraryType().getDescription()), //
+      Column.forString("Index(es)", LibraryDilutionSpreadSheets::listIndices), //
```

Comment:
@apmasell what do you think about also including targeted sequencing here?

Reply:
Probably useful...